### DISCUSSION TITLE
同时监听cell:change:* 和 cell:added，添加新cell时，只触发后者

### DISCUSSION THREAD

**Discussion by what2eat2night** [antvis/X6#2344] [Q&A]
我同时监听了 cell:change:* 和 cell:added
当我添加新的node 的时候，为什么 只有cell:added 被触发了

我想要时时保存 graph的修改 到 localStorage，
有没有更好的 监听变化的方法

**Comment by NewByVector** [comment]
这是两类事件，你可以还需要监听 `cell:removed`。

**Reply by what2eat2night** [reply]
能详细讲讲吗？ 有没有一个 总的地方 可以监听，这样我不用在每个不同的监听里面，去调用实时保存的函数了

**Reply by NewByVector** [reply]
你可以监听三个事件，节点添加、删除、属性修改，对应 `cell:added`、`cell:removed`、`cell:change:*`。
